docs/reference/pages/editing_api.rst

```diff
@@ -163,10 +163,15 @@ PageChooserPanel
             )
 
             content_panels = Page.content_panels + [
-                PageChooserPanel('related_page', 'demo.PublisherPage'),
+                PageChooserPanel('related_page', ['demo.PublisherPage', 'demo.EventPage']),
             ]
 
-    ``PageChooserPanel`` takes two arguments: a field name and an optional page type. Specifying a page type (in the form of an ``"appname.modelname"`` string) will filter the chooser to display only pages of that type.
+    ``PageChooserPanel`` takes two arguments: a field name and an optional page type.
```

Comment:
I think this documentation should be more explicit about the fact this parameter can be a list. As it is, it is just tacked on to the end, probably after most people have finished reading! Consider:

```rst
``PageChooserPanel`` takes two arguments: a field name and an optional list of page types.

The ``page_types`` parameter restricts which page types can be selected in this field. It defaults to just the model that the ``ForeignKey`` is pointing at. A ``ForeignKey`` pointing at ``wagtailcore.Page``, such as the one in the example above, will allow all pages to be selected, as all pages must subclass ``wagtailcore.Page``.

This parameter will take a list of Page subclasses, or strings of the form ``"app_name.ModelName"`` that name a Page subclass.